scripts/airgap/image-list.txt

```diff
@@ -1,5 +1,5 @@
-docker.io/rancher/klipper-helm:v0.6.6-build20211022
-docker.io/rancher/klipper-lb:v0.3.4
+docker.io/rancher/klipper-helm:v0.6.8-build20211123
+docker.io/rancher/klipper-lb:v0.3.5
 docker.io/rancher/local-path-provisioner:v0.0.21
 docker.io/rancher/mirrored-coredns-coredns:1.8.6
 docker.io/rancher/mirrored-library-busybox:1.34.1
```

Comment:
do we have s390x images for any of the mirrored images? I don't believe our mirroring script would copy those architectures, even if upstream did contain them.

Reply:
I can see it for mirror https://hub.docker.com/r/rancher/mirrored-library-busybox/tags. Also, I believe https://github.com/rancher/image-mirror/pull/175 PR should add mirroring for s390x as well.